src/api/spec/factories/attrib_namespaces.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :attrib_namespace do
-    name { Faker::Lorem.word }
+    name { Faker::Lorem.unique.word }
```

Comment:
JFYI, you could also use a [sequence](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences) here

Reply:
I don't think so. I believe the sessions are per suite - which make them even more dangerous to collision on test reruns.